website/docs/getting-started/setup/installation-guides/aws-ecs-on-fargate.md

```diff
@@ -87,36 +114,35 @@ Once the cluster is created, you need to create a task that runs on the cluster
 1. On the sidebar, choose Task Definitions and Create a new Task Definition.
 2. Choose Fargate as the launch type, and proceed to the next step.
 3. Enter the task definition name.
-4. Set the task role to None.
+4. Set the task role to the one created in prerequisites - **Create the ECS task role**.
 5. Select the default Network mode.
 6. Set Linux as the Operating system family.
-7. Set the Task Execution Role to the one created in the prerequisite step.
-8. Select the default Task execution IAM role created in the Prerequisite step.
-9. Set the required task size for memory & CPU (Minimum requirement: 2vCPU and 4 GB Memory).
-10. Go to the **Volumes** section and add a new volume. Enter the Name as `appsmith_stack`, set the Volume type as **EFS**, and set the **File System ID** to the EFS filesystem created in the prerequisite step. Leave the remaining fields with the default values.
+7. Set the Task Execution Role option to [**Create new role**].
+8. Set the required task size for memory & CPU (Minimum requirement: 2vCPU and 4 GB Memory).
+9. Go to the **Volumes** section and add a new volume. Enter the Name as `appsmith_stack`, set the Volume type as **EFS**, and set the **File System ID** to the EFS filesystem created in the prerequisite step. Leave the remaining fields with the default values.
 
-    ![EFS Volume Configuration](/img/ecs-efs.png)
+    ![EFS Volume Configuration](/img/ecs-efs-fargate.png)
 
-11. Configure **Appsmith container**.
+10. Configure the **Appsmith container**.
     1. Click the **Add container** button.
     2. Enter the container name, and set the Image to `appsmith/appsmith-ce`.
-    3. Add port mappings for the ports **80->80,443->443**.
+    3. Add port mappings for ports **80->80,443->443**.
     4. Set the _Mount points Source volume_ to `appsmith_stack` and set the Container path to `/appsmith-stacks`.
 
-    ![Storage Setting](/img/storage_settings_aws_fargate.png)
+    ![Storage Setting](/img/storage-settings-ecs-fargate-efs.png)
 
     5. You can configure the Environment Values for the Appsmith in the Environment Section. For sensitive values, it's **recommended** you create secrets and set the `env` value using the ValueFrom option by specifying the `arn` of the secret created.
```

Comment:
```suggestion
    5. You can configure the environment variable for Appsmith in the **Environment** Section. For sensitive values, it's **recommended** that you create secrets and set the `env` value using the **ValueFrom** option by specifying the `arn` of the secret created.
```

Reply:
@sum35h - Can we rename Environment Values to environment variables? I presume this is on AWS.